crds/roman/tpns/wfi_dark.tpn

```diff
@@ -13,4 +13,4 @@ ROMAN.META.INSTRUMENT.DETECTOR      H   C   R
 
 ROMAN.META.EXPOSURE.TYPE  H   C   R
 
-ROMAN.META.OBSERVATION.MA_TABLE_NAME    H   C   R
+ROMAN.META.OBSERVATION.MA_TABLE_NUMBER    H   C   R
```

Comment:
According to the schema `MA_TABLE_NUMBER` is of integer type. Should this be then ` H I R`?